01-cloudformation/1.3.2/deploys3.py

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/env python3
+import boto3
+import logging
+from botocore.exceptions import ClientError
+from regionslist import regions
+
+session = boto3.Session(profile_name='temp')
```

Comment:
https://twitter.com/ben11kehoe/status/1341378055319633920

Interesting discussion in the thread on opening a session vs getting a client directly. For this particular case there's not a right answer, but as I'm starting to work with more long-lived or more-frequently-invoked lambda functions, I see the value of using Sessions. I'd previously used them only when doing an AssumeRole.